evalml/model_understanding/visualizations.py

```diff
@@ -373,25 +374,44 @@ def get_prediction_vs_actual_over_time_data(pipeline, X, y, X_train, y_train, da
     dates = infer_feature_types(dates)
     prediction = pipeline.predict_in_sample(X, y, X_train=X_train, y_train=y_train)
 
-    return pd.DataFrame(
-        {
-            "dates": dates.reset_index(drop=True),
-            "target": y.reset_index(drop=True),
-            "prediction": prediction.reset_index(drop=True),
-        },
-    )
+    if is_multiseries(pipeline.problem_type):
+        return pd.DataFrame(
+            {
+                "dates": dates.reset_index(drop=True),
+                "target": y.reset_index(drop=True),
+                "prediction": prediction.reset_index(drop=True),
+                "series_id": X[pipeline.series_id].reset_index(drop=True),
+            },
+        )
+    else:
+        return pd.DataFrame(
+            {
+                "dates": dates.reset_index(drop=True),
+                "target": y.reset_index(drop=True),
+                "prediction": prediction.reset_index(drop=True),
+            },
+        )
 
 
-def graph_prediction_vs_actual_over_time(pipeline, X, y, X_train, y_train, dates):
+def graph_prediction_vs_actual_over_time(
+    pipeline,
+    X,
+    y,
+    X_train,
+    y_train,
+    dates,
+    single_series=None,
+):
     """Plot the target values and predictions against time on the x-axis.
 
     Args:
         pipeline (TimeSeriesRegressionPipeline): Fitted time series regression pipeline.
-        X (pd.DataFrame): Features used to generate new predictions.
-        y (pd.Series): Target values to compare predictions against.
+        X (pd.DataFrame): Features used to generate new predictions. If problem is multiseries, X should be stacked.
+        y (pd.Series): Target values to compare predictions against. If problem is multiseries, y should be stacked.
         X_train (pd.DataFrame): Data the pipeline was trained on.
         y_train (pd.Series): Target values for training data.
         dates (pd.Series): Dates corresponding to target values and predictions.
+        single_series (str): The single series that will be plotted from multiseries. Defaults to None
```

Comment:
Can you update this a little bit to make it clearer what this is referencing? i.e. along the lines of "a single series id value to plot just one series in a multiseries dataset"